ConsoleApp/InternetShop/ProductType.cs

```diff
@@ -0,0 +1,23 @@
+namespace ConsoleApp.InternetShop;
+
+enum ProductType
+{
+LaptopsAndComputers,
+Smartphones, 
+TV, 
+Electronics,
+GoodsForGamers,
+HouseholdAppliances,
+HomeGoods,
+ToolsAndAutoGoods,
+PlumbingAndRepair,
+gardenAndVegetableGarden,
```

Comment:
code convention violated here
```suggestion
GardenAndVegetableGarden,
```